### ISSUE TITLE
"Rails/ReversibleMigration incorrect flags reversible usage of change_column_default(without :from and :to)

### ISSUE THREAD

**Issue by rhys-vdw** [rubocop-hq/rubocop#6729]
## Expected behavior

Should ignore reversible usage of `change_column_default` in the following migration:

```rb
require_relative '../helpers/reversible_helpers'

class AddIsRedirectEnabledFlagToSets < ActiveRecord::Migration[5.1]
  include ReversibleHelpers

  def change
    column = [:test_sets, :is_redirect_enabled]
    add_column(*column, :boolean)
    up_only do
      execute <<~SQL
        update #{column[0]}
        set #{column[1]} = not (
          redirect_link is null
          or trim(redirect_link) = ''
        )
      SQL
    end
    change_column_null(*column, false)
    change_column_default(*column, from: nil, to: false)
  end
end
```

## Actual behavior

```
db/migrate/20190129042815_add_is_redirect_enabled_flag_to_sets.rb:19:5: C: Rails/ReversibleMigration: change_column_default(without :from and :to) is not reversible.
    change_column_default(*column, from: nil, to: false)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

Check the above migration. You can remove the `require_relative` and `up_only` statements.

## RuboCop version

```
-> bundle exec rubocop -V
0.62.0 (using Parser 2.5.3.0, running on ruby 2.4.5 x86_64-darwin17)
```

Not latest, sorry. I couldn't find any recent related issues or PRs so I assume it's not fixed. I'm not in a position to upgrade just now because it takes a long time to deal with all the rule changes.

**Comment by rhys-vdw** [comment]
Is rubocop still being maintained? There's a PR for this issue here but it's being marked as stale? Why mark issues as stale if PRs are going unmerged? https://github.com/rubocop-hq/rubocop/pull/6820
